### PR TITLE
Issue327 com tim rat

### PR DESCRIPTION
**Pull request by dhblum**
This is for #327.

@JavierArroyoBastida After you review, I'll update reference results.  Please also let me know if it is worth keeping the plot functionality of ``kpi_calculator.get_computational_time_ratio()``, as it would need to be fixed to use a vector of stored simulation steps, rather than assuming constant throughout the test period.

### CODE REVIEW

File: testcase.py [hunk 1]
```diff
@@ -566,7 +566,7 @@ def get_name(self):
 
         return name
 
-    def get_elapsed_control_time(self):
+    def get_elapsed_control_time_ratio(self):
         '''Returns the elapsed control time vector for the case.
```
Comment (JavierArroyoBastida) [inline testcase.py:570]:
```suggestion
        '''Returns the elapsed control time ratio vector for the case.
```

File: testing/test_testcase1.py [hunk 2]
```diff
@@ -277,6 +278,33 @@ def test_return(self):
...
+
+    def test_kpi(self):
+        '''Tests the calculation of the kpi.
+
+        '''
+
+        # Run test
+        requests.put('{0}/initialize'.format(self.url), data={'start_time':0, 'warmup_period':0})
+        step = requests.get('{0}/step'.format(self.url)).json()
+        for i in range(5):
```
Comment (JavierArroyoBastida) [inline testing/test_testcase1.py:301]: Should we test that we can also handle variable control steps for computational time ratio? I mean, we could set a variable step as function of the `i` index either in this same test or [here](https://github.com/ibpsa/project1-boptest/blob/bf9d49873ec3b475c9324092141628b32ea9c785/testing/test_kpis.py#L131).

**Review by JavierArroyoBastida — COMMENTED** [review]
@dhblum this is looking good, thanks for the implementation. Just added two suggestions.

File: testing/test_testcase1.py (hunk 2 above)
Comment (dhblum) [reply testing/test_testcase1.py:301]: Yes, good point, thanks.  I can implement.

Comment (dhblum) [reply testing/test_testcase1.py:301]: Added in https://github.com/ibpsa/project1-boptest/pull/329/commits/d470b43388002cb96568d2f416154736f60e8a25.

Comment (JavierArroyoBastida) [reply testing/test_testcase1.py:301]: Thanks @dhblum! looking good.

**Review by JavierArroyoBastida — APPROVED** [review]

**Pull request merged**